src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -69,17 +75,78 @@ export class FeaturedArtists extends React.Component<FeaturedArtistsProps, Featu
     return allArtists
   }
 
+  failedFollowChange() {
+    // callback for analytics purposes
+    this.setState({
+      isFollowedChanging: false,
+    })
+  }
+
+  successfulFollowChange() {
+    // callback for analytics purposes
+    Events.userHadMeaningfulInteraction()
```

Comment:
Out of curiosity, I see this is related to requesting App Store reviews — any  advice on what constitutes a "meaningful interaction"?